docs/man/git-lfs-ext.1.ronn

```diff
@@ -0,0 +1,25 @@
+git-lfs-ext(1) - View extension details
+==============================================================
+
+## SYNOPSIS
+
+`git lfs ext list` [<name>...]
+
+## DESCRIPTION
+
+Git LFS extensions enable the manipulation of files streams
+during smudge and clean.
+
+## EXAMPLES
+
+* List details for all extensions
+
+    `git lfs ext list`
+
+* List details for the specified extensions
+
+    `git lfs ext list 'foo' 'bar'`
+
```

Comment:
That's correct, both `git lfs ext` and `git lfs ext list` show details for all of the extensions.  I'll make that more clear in the documentation.